requirements.txt

```diff
@@ -2,3 +2,4 @@ six>=1.9.0
 requests
 django==1.10
 netCDF4
+Pydap==3.2.0
```

Comment:
There's already a pydap requirement in `setup.py` so this should go there to avoid future confusion:

https://github.com/Applied-GeoSolutions/gips/blob/dev/setup.py#L62

I think I made a mistake in introducing `requirements.txt` as it introduces an extra place to put this information.  I should've learned how to use `setuptools` the right way instead.